lib/perl/Genome/Model/Tools/Speedseq/Base.pm

```diff
@@ -128,7 +128,8 @@ sub _tool_input_file_metas {
 # formatting (argname=argvalue).
 sub _format_tool_arg {
     my ($type, $name, $value) = @_;
-    return '-'. $name if ($type eq 'Boolean');
+    return '-'. $name if ($type eq 'Boolean' and $value);
+    return "" if ($type eq 'Boolean');
```

Comment:
It might be nice to group the Boolean handling in a block or subroutine instead of checking the condition at the end of each line to which it applies.